frontend-ssr/components/search/Search.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import _ from 'lodash';
```

Comment:
Let's use the `lodash-es` module and import just what we need:

`import { debounce } from 'lodash-es'`.

This way we only add to the final bundle the code we need, not all of lodash which is huge - https://bundlephobia.com/result?p=lodash@4.17.20